### ISSUE TITLE
Pick another file, if there is no needed version

### ISSUE THREAD

**Issue by Alex0007** [bahmutov/pick-precompiled#2]
For example we haven't compiled file for our Node version

Can we somehow pick file from previous version and not to throw error?

`Error: could not pick bundle for node 5.5.0 among array with 4 items.
[".build/index.compiled.for.0.10.js",".build/index.compiled.for.0.11.js",".build/index.compiled.for.0.12.js",".build/index.compiled.for.4.js"]`

**Comment by bahmutov** [comment]
Good idea, the search / match is very very raw right now, it should semantically order available versions and just pick the next lowest version
